src/alto-ui/Form/DatePicker/DatePicker.js

```diff
@@ -122,6 +123,12 @@ function DatePicker(props) {
     <>
       <TextField
         {...remainingProps}
+        onKeyDown={e => {
+          if (e.key === 'enter') {
```

Comment:
this will **fail** 💥 , On Chrome key is "Enter" with upper case, same IE
remember to add `toLoserCase()` if you want to compare with  `===` the strings

Reply:
true, I suggested by /components/DatagridCellInput/DatagridCellInput.js
there is e.key and I use here similarly.
I changed also in DatePicker.js and DatagridCellInput.js